spec/core/ics-004-channel-and-packet-semantics/UPGRADES.md

```diff
@@ -571,16 +571,16 @@ function chanUpgradeTry(
   }
 
   // NON CROSSING HELLO CASE:
-  // if the counterparty sequence is less than the current sequence,
```

Comment:
I think this was correct because in the non-crossing hello case the `counterpartyUpgradeSequence` should be `==` to `expectedUpgradeSequence` (which is `channel.UpgradeSequence + 1` on line 570).

Reply:
@crodriguezvega Thank you for reviewing this!

> in the non-crossing hello case the counterpartyUpgradeSequence should be == to expectedUpgradeSequence (which is channel.UpgradeSequence + 1 on line 570)

Yes, but doesn't `the current sequence` mean not `expectedUpgradeSequence` but `channel.UpgradeSequence`?

If so, I think that:
`counterpartyUpgradeSequence < expectedUpgradeSequence`
<=> `the counterparty sequence is less than (the current sequence + 1)`
<=> `the counterparty sequence is less than or equal to the current sequence`

Reply:
Sorry for taking so long to come back to this... Yes, I think you're right! Thank you for explaining and correcting me. 🙏